### PR TITLE
Malins happy thoughts

### PR DESCRIPTION
**Pull request by malin798**
Link to my deployed project:

https://malins-project-happy-thoughts.netlify.com/

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/Components/LikePost.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+       })
+  
+  }, []);
+  
+  return (
+    <>
+      <button className="emoji"  aria-label="heart emoji" 
+      onClick={() => setLikes(likes +1)} 
+      style={likes === 0 ? {backgroundColor: '#ccc9c9'} : {backgroundColor: '#ffb2b2'}}
+      >
```
Comment (ericaglimsholt) [inline code/src/Components/LikePost.js]: Suggestion of structure:
```suggestion
      <button 
        className="emoji"  
        aria-label="heart emoji" 
        onClick={() => setLikes(likes +1)} 
        style={likes === 0 ? {backgroundColor: '#ccc9c9'} : {backgroundColor: '#ffb2b2'}}
      >
```

File: code/src/Components/PostThoughts.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        "message": thought
+      })
+    })
+    .then(() => {
+      window.location.reload()
+    })
+   }
+
+  return (
+    <li className="my-thought">
```
Comment (ericaglimsholt) [inline code/src/Components/PostThoughts.js]: How come you are using `<ul>` and `<li>` for this? :)

File: code/src/Components/PostThoughts.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      <form onSubmit={handleSubmit}> 
+
+        <h2>What is making you happy right now?</h2>
+
+        <textarea 
+        placeholder="Fill in your happy thought here"
+        maxLength="140"
+        minLength="5"
+        value={thought}
+        onChange={event => setThought(event.target.value)}>
```
Comment (ericaglimsholt) [inline code/src/Components/PostThoughts.js]: Indent one step would help with the structure

**Comment by malin798** [comment]
> Great work Malin!
> Something I notice is that if you update the page the likes will add +1 to them. Have you seen it? I'll still approve this assignment, but it's something for you to look at if you want to add this to your portfolio later on :)

Thank you! I had not noticed!! I fixed it right away, I forgot to do the fetch onClick....

**Pull request closed without merging**